openvpn-install.sh

```diff
@@ -214,9 +214,16 @@ function installQuestions() {
 	echo "I need to ask you a few questions before starting the setup."
 	echo "You can leave the default options and just press enter if you are ok with them."
 	echo ""
+
+	echo "Please provide the VPN network prefix. If you are going to use  10.8.0.0/24, then type prefix 10.8.0"
+	until [[ $VPN_NETWORK =~ ^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){2}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$ ]]; do
+		echo "Format bust be like XXX.XXX.XXX"
+		VPN_NETWORK=${VPN_NETWORK:-10.8.0}
```

Comment:
What is this line for? You can write `10.8.0` in the `read -i` command as it is.

Can you move it to `installOpenVPN()` instead, it's needed for the auto-install.

Reply:
This is a bug. moving to `installOpenVPN()` as suggested. thanks